### PR TITLE
Modular

### PR DESCRIPTION
**Pull request by Toxici11i**

### CODE REVIEW

**Comment by Toxici11i** [comment]
this is doing weird conflcit shit, dont forcemerge, instead check each commit and take each changed file

File: code/modules/SCPmodular/jobprocs.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+	var/mob/living/scp_173/A = new /mob/living/scp_173 (loc)
+	A.a_intent = I_HURT
+	A.key = key
+
+	to_chat(A, "<B>You are now 173. Snap Snap.</B>")
+	qdel(src)
+	return
+
+/mob/living/carbon/human/proc/spcize()
```
Comment (mattmatterson111) [inline code/modules/SCPmodular/jobprocs.dm:23]: We should look into giving these better names, once this gets merged.

File: code/modules/projectiles/guns/energy/nuclear.dm [hunk 2]
```diff
@@ -85,8 +85,7 @@
 //override for failcheck behaviour
 /obj/item/weapon/gun/energy/gun/nuclear/Process()
 	if(fail_counter > 0)
-		SSradiation.radiate(src, (fail_counter * 2))
-		fail_counter--
+		SSradiation.radiate(src, fail_counter--)
```
Comment (mattmatterson111) [inline code/modules/projectiles/guns/energy/nuclear.dm:88]: I'm not sure this is an intended change.

File: config/example/config.txt [hunk 3]
```diff
@@ -1,420 +1,397 @@
-## Server name: This appears at the top of the screen in-game. In this case it will read "tgstation: station_name" where station_name is the randomly generated name of the station for the round. Remove the # infront of SERVERNAME and replace 'tgstation' with the name of your choice
```
Comment (mattmatterson111) [inline config/example/config.txt:1]: This is going to break some things. You've accidentally removed some of Bay's new configs.

Comment (Toxici11i) [reply config/example/config.txt:1]: no surprise there, gimme a sec to actually recommit and ill fix this

**Pull request closed without merging**